tensorflow_encrypted/io.py

```diff
@@ -21,11 +23,11 @@ def provide_input(self) -> tf.Tensor:
 
 class OutputReceiver(object):
 
-    def __init__(self, player: Player, output_fn: Optional[Callable[[tf.Tensor], tf.Tensor]] = None) -> None:
+    def __init__(self, player: Player, output_fn: OutputFunc = None) -> None:
         self.player = player
         self.output_fn = output_fn
 
-    def receive_output(self, tensor: tf.Tensor) -> tf.Tensor:
+    def receive_output(self, tensor: Union[tf.Tensor, List[tf.Tensor]]) -> tf.Tensor:
```

Comment:
could you change the return type to `tf.Operation` btw? returning `tf.Tensor` is a mistake (https://github.com/mortendahl/tf-encrypted/blame/master/tensorflow_encrypted/io.py)